Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -227,6 +227,15 @@ This token sets the bitrate control encoding mode [1: Variable Bitrate, 0: Const
 
 This token sets the encoder to automatically choose the best quality encoding mode that allows the encoder to run at a real-time speed set by the –fps parameter (described in the next section).
 
+>-lp integer **[Optional]**
+
+This token sets the number of logical processors which the encoder threads run on.
+
+>-ss integer **[Optional]**
+
+For dual socket systems, this token specific which socket the encoder runs on.
+
```

Comment:
I meant add the y4m usage here, not -lp and -ss.

Reply:
I don't think -lp -ss are needed here, since there are so many other optional parameters.